### PR TITLE
UGENE-5640 Strict consensus algorithm draws an empty space instead of gap if a threshold is too low

### PR DESCRIPTION
**Pull request by DmitriiSukhomlinov**

### CODE REVIEW

File: src/corelibs/U2Algorithm/src/util_msa_consensus/MSAConsensusAlgorithmStrict.cpp [hunk 1]
```diff
@@ -58,8 +58,8 @@ char MSAConsensusAlgorithmStrict::getConsensusChar(const MultipleAlignment& ma,
     // use gap is top char frequency is lower than threshold
     int nSeq = (seqIdx.isEmpty() ? ma->getRowCount() : seqIdx.size());
     int currentThreshold = getThreshold();
-    int cntToUseGap = int(currentThreshold / 100.0 * nSeq);
-    int topFreq = freqsByChar[topChar];
+    double cntToUseGap = double(currentThreshold) / 100.0 * nSeq;
```
Comment (yalgaer) [inline src/corelibs/U2Algorithm/src/util_msa_consensus/MSAConsensusAlgorithmStrict.cpp]: Cast to double is not needed.

**Review by yalgaer — CHANGES_REQUESTED** [review]
Please fix the xml test

File: src/corelibs/U2Algorithm/src/util_msa_consensus/MSAConsensusAlgorithmStrict.cpp (hunk 1 above)
Comment (DmitriiSukhomlinov) [reply src/corelibs/U2Algorithm/src/util_msa_consensus/MSAConsensusAlgorithmStrict.cpp]: OK

**Comment by DmitriiSukhomlinov** [comment]
I checked an XML test. The result has been changed a bit. This has happened because since I removed the `int` rounding numbers became more accurate. For example, it was a comprising before the fix:
```
int cntToUseGap = int(currentThreshold / 100.0 * nSeq); // 12
int topFreq = freqsByChar[topChar]; // 12
char res = topFreq < cntToUseGap ? U2Msa::GAP_CHAR : (char)topChar; // 12 < 12 => false => (char)topChar
```

But, after the fix it was turned into a:
```
double cntToUseGap = currentThreshold / 100.0 * nSeq; //12.6
double topFreq = freqsByChar[topChar]; // 12.0
char res = topFreq < cntToUseGap ? U2Msa::GAP_CHAR : (char)topChar; // 12.0 < 12.6 => true => U2Msa::GAP_CHAR
```

**Review by yalgaer — APPROVED** [review]

**Pull request merged**